lmdeploy/model.py

```diff
@@ -1013,6 +1013,57 @@ def messages2prompt(self, messages, sequence_start=True):
         return ret
 
 
+@MODELS.register_module(name=['deepseek-chat'])
+class Deepseek(BaseModel):
+
+    def __init__(self, user='User:', assistant='Assistant:', **kwargs):
+        super().__init__(**kwargs)
+        self._user = user
+        self._assistant = assistant
+        self._bos = '<｜begin▁of▁sentence｜>'
+
+    def decorate_prompt(self, prompt, sequence_start=True):
+        """Return the prompt that is concatenated with other elements in the
+        chat template.
+
+        Args:
+            prompt (str): the input prompt
+            sequence_start (bool): indicator for the first round chat of a
+               session sequence
+        Returns:
+            str: the concatenated prompt
+        """
+        ret = f'{self._user} {prompt}\n\n{self._assistant}'
+        if sequence_start:
+            ret = f'{self._bos}{ret}'
+        return ret
+
+    def messages2prompt(self, messages, sequence_start=True):
```

Comment:
outputs always starts with an empty space, is there something wrong with chat template?
```
Response(text=' Hello! How can I assist you today?', generate_token_len=9, input_token_len=9, session_id=0, finish_reason='stop'), 
Response(text=' Yes, this is a test. Is there anything specific you would like me to test or do for you?', generate_token_len=22, input_token_len=13, session_id=1, finish_reason='stop')
```